trio/_core/_parking_lot.py

```diff
@@ -79,12 +79,13 @@
 import attr
 
 from .. import _core
-from .._util import Final
+from .._util import final
 
 if TYPE_CHECKING:
     from ._run import Task
 
 
+@final
```

Comment:
Just noting that this adds a `@final` where there were none before. It might have been nicer to do so in another PR for a nicer diff but eh, not a priority. Up to you.

Reply:
Should probably revert these two, `test_exports` currently just allows the stats classes to be final.